source/manual/browser-support.html.md

```diff
@@ -5,8 +5,12 @@ layout: manual_layout
 section: Frontend
 type: learn
 owner_slack: "#govuk-frontenders"
-last_reviewed_on: 2018-08-29
+last_reviewed_on: 2019-10-08
 review_in: 12 months
 ---
 
-For browser support, see the Service Manual page about [Designing for different browsers and devices](https://www.gov.uk/service-manual/technology/designing-for-different-browsers-and-devices).
+GOV.UK [shares the same browser support matrix with GOV.UK Frontend](https://github.com/alphagov/govuk-frontend#browser-support) for pages served to the general public.
+
+If the application you are working on is aimed at internal users rather than the general public you should look at your analytics data and check which browsers your users are using. Then you can then make an informed decision about which browsers to test with.
```

Comment:
👍

Reply:
I wonder if this line is valuable as a general point? Look at your traffic and decide which browsers to support (regardless of whether your users are internal or not)